anonlink/solving/_multiparty_solving_inner.cpp

```diff
@@ -170,7 +170,7 @@ class EdgesMatrix {
     CountType increment(T key0, T key1) {
         // Call set_or_increment twice for symmetry.
         CountType retval0 = set_or_increment(key0, key1, 1);
-        CountType retval1 = set_or_increment(key1, key0, 1);
+        CountType retval1[[maybe_unused]] = set_or_increment(key1, key0, 1);
```

Comment:
NB: the warning this silences was there before, but I thought this was a good opportunity to fix it.